bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/util/NumberField.scala

```diff
@@ -39,20 +38,22 @@ class NumberField(d: Option[Double]) extends FormattedTextField(NumberField.df)
   reactions += {
     case ValueChanged(_) =>
 
-      valid = try {
-        val pp = new ParsePosition(0)
-        valid(NumberField.df.parse(text, pp).doubleValue) && (pp.getIndex == text.length())
-      } catch {
-        case _:Exception => false
+      // REL-2253 For some reason on OSX you get sometimes an extra value change event with an empty text.
+      // This creates a null parsed value and the validation fails
+      // This is not very noticeable as there is a second event with the valid text
+      // However, it means that sometimes you need to press Ok twice
```

Comment:
What happens now if you delete all the text, which presumably legitimately produces an event with empty text?